corehq/apps/export/tasks.py

```diff
@@ -211,3 +210,8 @@ def generate_schema_for_all_builds(self, schema_cls, domain, app_id, identifier)
         only_process_current_builds=False,
         task=self,
     )
+
+
+@periodic_task(run_every=crontab(hour="23", minute="59", day_of_week="*"))
+def delete_expired_datafile_blobs():
+    DataFile.objects.filter(delete_after__lte=datetime.utcnow()).delete()
```

Comment:
This looks like it's deleting the SQL objects, but not the actual blobs, which means the blobs will be orphaned in the blob DB.

Reply:
Oh! You are right. I was under the impression that calling `delete` on a queryset would use the overridden `delete` method from the model, which isn't true. Good catch!
https://github.com/dimagi/commcare-hq/pull/21562/commits/6ca0b2601d19470ce183a2a3769a10192634663d